src/screens/NationDetailsContainer/index.js

```diff
@@ -128,7 +128,10 @@ NationDetailsContainer.PropTypes = {
 const mapStateToProps = state => ({
   ...state.nations,
   ...state.wallet,
-  isDraft: isDraft(openedNation(state.nations) || {}),
+  isDraft: (() => {
+    const nation = openedNation(state.nations);
+    return nation ? isDraft(nation) : true;
```

Comment:
Can we use a explicit check here as well? like `typeof nation === 'object'`

Reply:
I use nation !== null to check.
Important that, typeof nation === 'object' is TRUE if nation is null.

Reply:
Haha nice. Good catch.